.travis.yml

```diff
@@ -1,24 +1,51 @@
 # Copyright (c) 2015-2016, NVIDIA CORPORATION.  All rights reserved.
 
-sudo: required
+os: linux
 dist: trusty
-
+sudo: required
 language: python
 python: 2.7
 
 env:
+    global:
+        - CAFFE_ROOT=~/caffe
+        - TORCH_ROOT=~/torch
+        # Fixes for Torch and OpenBLAS
+        - OMP_NUM_THREADS=1
+        - OPENBLAS_MAIN_FREE=1
     matrix:
         - DIGITS_TEST_FRAMEWORK=caffe CAFFE_FORK=NVIDIA
         - DIGITS_TEST_FRAMEWORK=caffe CAFFE_FORK=BVLC
         - DIGITS_TEST_FRAMEWORK=torch
         - DIGITS_TEST_FRAMEWORK=none
-        - DEB_BUILD=true
+
+matrix:
```

Comment:
It's a little funky. We create a matrix with `env: matrix` that is our standard build matrix. If we were to change `python: 2.7` to `python: 2.7, 3.5` then we would have 8 standard build targets (2x4). Then, after this standard matrix is constructed, we can "include" or "exclude" some specific targets. I'm using it to include `LINT_CHECK` and `DEB_BUILD`, but we could also exclude "python3 and FRAMEWORK=tensorflow" or something else that was part of the standard matrix.